templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
Yeah, it's a bit of dogfooding, the idea is if somebody reads through this template trying to see what's going on, they can see how ELB's are commonly created in the k8s side of things.  But it's a bit hackish though because we wait in a loop for it to finish and there's no recourse if something went horribly wrong.

I think if we do a WaitConditionHandle it can at least block the stack from finishing, and we can do a reasonable timeout.

One thing I did notice is that the logs being output by this script (it has a -v flag after all) are available over CloudWatch thanks to the integration we have here, so we can have people look at the CoudWatch logs if they need to see what went wrong.